compiler-cli/src/main.rs

```diff
@@ -231,15 +231,22 @@ enum Command {
     Export(ExportTarget),
 }
 
-#[derive(Subcommand, Debug, Clone, Copy)]
+#[derive(Subcommand, Debug, Clone)]
 pub enum ExportTarget {
-    /// Precompiled Erlang, suitable for deployment.
+    /// Precompiled Erlang, suitable for deployment
     ErlangShipment,
-    /// The package bundled into a tarball, suitable for publishing to Hex.
+    /// The package bundled into a tarball, suitable for publishing to Hex
     HexTarball,
-    /// The JavaScript prelude module.
+    /// The JavaScript prelude module
     JavascriptPrelude,
+    /// The TypeScript prelude module
     TypescriptPrelude,
+    /// The package json interface
+    PackageInterface {
+        #[clap(long = "out", required = true)]
+        /// A file to write the json interface to, relative to the package root
```

Comment:
```suggestion
        /// The path to write the JSON file to
```
Might not be relative